src/app/organizations/settings/organization-subscription.component.ts

```diff
@@ -213,4 +214,14 @@ export class OrganizationSubscriptionComponent implements OnInit {
         return (this.sub.planType !== PlanType.Free && this.subscription == null) ||
             (this.subscription != null && !this.subscription.cancelled);
     }
+
+    get subscriptionDesc() {
+        if (this.sub.maxAutoscaleSeats == this.sub.seats && this.sub.seats != null) {
+            return this.i18nService.t('subscriptionMaxReached', this.sub.seats.toString());
+        } else if (this.sub.maxAutoscaleSeats == null) {
```

Comment:
`else if` doesn't do anything here. I think we just replace it with `if`. And remove the `else` below as well.

Reply:
This `else if` is hit if `maxAutoscaleSeats == null` and `seats ≠ null`. It's if your organization has a seat limit, but you haven't defined a max seat limit.

These are the three possibilities:
1. **`maxAutoscaleSeats == seats && != null**: We've reached the maximum limit of seats, Indicate we can no longer expand
2. **maxAutoscaleSeats == null**: Indicate we will expand indefinitely
3. **maxAutoscaleSeats != null && != seats**: Indicate we will expand until we hit max seats.

Reply:
Sorry for being vague, I meant that using `else if` and `else` in statements that returns values are unnecessary. And that we can just write

```typescript
if (this.sub.maxAutoscaleSeats == this.sub.seats && this.sub.seats != null) {
	return this.i18nService.t('subscriptionMaxReached', this.sub.seats.toString());
}

if (this.sub.maxAutoscaleSeats == null) {
    return this.i18nService.t('subscriptionUserSeatsUnlimitedAutoscale');
}

return this.i18nService.t('subscriptionUserSeatsLimitedAutoscale', this.sub.maxAutoscaleSeats.toString());
```